tools/codegen/codespec/model.go

```diff
@@ -83,8 +80,8 @@ type SetNestedAttribute struct {
 	NestedObject NestedAttributeObject
 }
 type SingleNestedAttribute struct {
-	Default    *CustomDefault
-	Attributes Attributes
+	Default      *CustomDefault
+	NestedObject NestedAttributeObject
```

Comment:
To be consistent with other nested attribute models (List and Map)

Reply:
In Tf schemas, `SingleNestedAttribute` actually define `Attributes: map[string]schema.Attribute{}` and not NestedObject
Do you still want to use NestedObject here?